src/store/user/userAction.js

```diff
@@ -24,11 +24,23 @@ export const updateUser = data => {
   const token = localStorage.getItem('authToken')
   return async dispatch => {
     try {
-      await API.updateUser(data, id, token)
-      dispatch({
-        type: types.UPDATE_USER,
-        user: data,
-      })
+      if (
+        data.nickname ||
```

Comment:
Hmm what is this for?

Reply:
Ah, I think this is left over from the "previous" version of checking whether the input by the user in profile creation is valid, but it's not necessary any more. I changed it back to the way it was.